src/main/java/io/confluent/connect/elasticsearch/bulk/BulkProcessor.java

```diff
@@ -395,6 +395,12 @@ private BulkResponse execute() throws Exception {
                       + "will attempt retry after {} ms. Failure reason: {}",
                       batchId, batch.size(), attempts, maxAttempts, sleepTimeMs, e.getMessage());
             time.sleep(sleepTimeMs);
+            if (Thread.currentThread().isInterrupted()) {
+              log.error(
```

Comment:
clear the interrupted flag with `Thread.interrupted()` here?

Reply:
why is this is a error log message?

Reply:
hmm never mind.

Reply:
I think error level is justified here because we failed to deliver the batches due to interrupt. Since we log error bellow if we fail to deliver after exhausting retries, it seems appropriate to log at error level here too.